analysis/deathknight/src/DeathsDue.js

```diff
@@ -0,0 +1,244 @@
+import { t } from '@lingui/macro';
```

Comment:
Please consider writing new modules in TypeScript. We've been migrating slowly to TypeScript for many benefits, and I reckon it would really enhance your coding experience and is worth the trouble of learning.